ssas/systems.go

```diff
@@ -660,6 +687,40 @@ func (system *System) ResetSecret(trackingID string) (Credentials, error) {
 	return creds, nil
 }
 
+// activeCredsCount returns the number of credentials (secrets) that are currently valid for the provided group ID
+func activeCredsCount(groupID string, ignoredClientID string) (int, error) {
```

Comment:
What is `ignoredClientID` for?  I see that it is only relevant when calling `ResetSecret()`, but I think I'm missing what its used for.

Reply:
I've refactored that function; hopefully the usage is clearer now.